TREnvironmentEditor/Model/Types/Mirroring/EMMirrorFunction.cs

```diff
@@ -944,12 +945,19 @@ private static bool AreDoubleDoors<T>(TREntity<T> door1, TREntity<T> door2)
         where T : Enum
     {
         // If the difference between X or Z position is one sector size, they share the same Y val,
-        // and they are facing the same diretion, then they're double doors.
-        return !EqualityComparer<T>.Default.Equals(door1.TypeID, door2.TypeID)
-            && door1.Room == door2.Room
-            && door1.Y == door2.Y
-            && door1.Angle == door2.Angle
-            && (Math.Abs(door1.X - door2.X) == SectorSize || Math.Abs(door1.Z - door2.Z) == SectorSize);
+        // and they are facing the same direction, then they're double doors.
+        if (EqualityComparer<T>.Default.Equals(door1.TypeID, door2.TypeID)
```

Comment:
Should there be an ! for Equals? Does it return a bool or an int?

Reply:
It's a bool, the conditions have been inverted so dropped the !. We only target dissimilar doors like at the end of caves. The ones at the start of Atlantis for example are ignored (although there would be no real harm in flipping them).